.konchrc

```diff
@@ -1,22 +1,19 @@
+#!/opt/conda/bin/python
```

Comment:
`.konchrc` should not have shebang lines, since they aren't executed as scripts. Please remove this.

Reply:
Understood.  It was added because it allows VS Code to recognize it as Python code since it does not have a .py extension.  Probably just trying to be unnecessarily clever.